doc/parameters/param_list.md

```diff
@@ -479,6 +479,8 @@ When `planner_plugins` parameter is not overridden, the following default plugin
 | ----------| --------| ------------|
 | stop_on_failure | true | Whether to fail action task if a single waypoint fails. If false, will continue to next waypoint. |
 | loop_rate | 20 | Rate to check for results from current navigation task |
+| sleep_time_inbetween_waypoints | 5 | Amount of time in seconds for robot to sleep/wait after each waypoint is reached. If zero, robot will directly continue to next waypoint. |
```

Comment:
Also must be added to the waypoint follower entry on navigation.ros.org + migration guides https://github.com/ros-planning/navigation.ros.org/blob/master/configuration/packages/configuring-waypoint-follower.rst

Reply:
Default must also be `0` to retain same behavior.